packages/server/src/api/storage/sql/sql.js

```diff
@@ -80,6 +80,7 @@ function createSequelize(options) {
   const sequelizeOptions = {
     operatorsAliases: false,
     logging: () => {},
+    ...options.sequelizeOptions,
```

Comment:
let's at least move `operatorsAliases: false` down below this so it can't be overriden, we don't want folks to enable that and all of the work has already been done in LHCI to make due without them

Reply:
Good call. I wasn't sure which settings you specifically wouldn't want configurable. What about `logging` does it make sense to leave that overridable?

Reply:
`logging` is fine with me, it's just unnecessary noise but if folks want more insight into what it's doing that's fine with me :)